packages/split-button/stories/index.ts

```diff
@@ -156,6 +157,7 @@ export const splitbutton = (
         type=${properties.type || 'field'}
         ?open=${!!properties.open}
         ?disabled=${properties.disabled}
+        description=${properties.description}
```

Comment:
please use a ifDefined here too

Reply:
here description is an inherited property, it's inherited from `picker` hence I believe we don't explicitly need `ifDefined`